### ISSUE TITLE
DropdownV2 not adding option roles

### ISSUE THREAD

**Issue by ghost** [carbon-design-system/carbon#2693]
## Environment

> Operating system
Mac osX

> Browser
Chrome 74.0

> Automated testing tool and ruleset
DAP
Ruleset: IBM Accessibility February 2019

## Detailed description

> What version of the Carbon Design System are you using?

6.113.1

> What did you expect to happen?

No potential violations for roles

> What happened instead?

```An element with WAI-ARIA role listbox does not contain or own at least one child element with each of the following WAI-ARIA roles: option. See rule help for remediation guidance.```
 
> What WCAG 2.1 checkpoint does the issue violate?

Checkpoint 4.1.2 Name, Role, Value

## Steps to reproduce the issue

1. Use <DropdownV2> on a page

> Please create a reduced test case in CodeSandbox
> https://codesandbox.io/s/github/IBM/carbon-components-react/tree/master/examples/codesandbox

https://codesandbox.io/s/6lq995w6rr
